iroha-cli/interactive/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Interactive Client
+add_library(interactive_cli
+    impl/interactive_cli.cpp
+    impl/interactive_common_cli.cpp
+    impl/interactive_query_cli.cpp
+    impl/interactive_transaction_cli.cpp
+    )
+target_link_libraries(interactive_cli
+    model
+    parser
+    )
+target_include_directories(interactive_cli PUBLIC
+    ${PROJECT_SOURCE_DIR}/iroha-cli
+    )
```

Comment:
Add endline, and also set up your IDE to add endlines automatically.

Reply:
Fixed. Thanks